### PR TITLE
Narrow north star question screenshot tests to only question itself

### LINKED ISSUE #7533: [North Star Applicant UI] Update question type screenshot goldens to only contain question itself
Right now, all north star screenshot tests cover the whole page, which means a small change to a header or footer means we have to update every single screenshot. We should modularize them and have question type screenshots just cover the question element itself

### PR DESCRIPTION
**Pull request by greg-hyde**
### Description

When doing screenshot tests for question types in the north star ui, only screenshot the question element itself so we don't need to handle diffs from changes outside of those elements. There are other tests that cover the full page.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [x] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)
- [x] Created unit and/or browser tests which fail without the change (if possible)

### Issue(s) this completes

Fixes #7533

### CODE REVIEW

**Comment by greg-hyde** [comment]
* **#7537** <a href="https://app.graphite.dev/github/pr/civiform/civiform/7537?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @greg-hyde and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by rockycodes — COMMENTED** [review]
Lgtm! This is a great change.

File: server/app/views/questiontypes/StaticTextQuestionFragment.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div th:fragment="static-text(question)">
+<div th:fragment="static-text(question)" data-testid="staticQuestionRoot">
```
Comment (avaleske) [inline server/app/views/questiontypes/StaticTextQuestionFragment.html:1]: _very_ nit but should this be `questionRoot` to match the others?

Comment (avaleske) [reply server/app/views/questiontypes/StaticTextQuestionFragment.html:1]: ohh it's cause there's two questions on the page, isn't it?

**Review by avaleske — APPROVED** [review]
thank you!

We still have a screenshot of the overall question page, right?

File: server/app/views/questiontypes/StaticTextQuestionFragment.html (hunk 1 above)
Comment (greg-hyde) [reply server/app/views/questiontypes/StaticTextQuestionFragment.html:1]: yep I had to give it a different Id because there are two questions

**Pull request merged**